sources/packages/backend/apps/workers/src/types/max-jobs-to-activate.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Suggested values to be considered while defining the
+ * amount of jobs that a worker can request to be activated.
+ * !The best choice for every scenario can be defined only after
+ * !some benchmarks are acquired. The below options and descriptions
+ * !are just a suggestion to have a start point. As a general recommendation
+ * !activating one job per call does not seems a recommended practice.
+ */
+export enum MaxJobsToActivate {
```

Comment:
mm... how do we decide the count for a worker?

Reply:
Nothing too concrete right now. Without executing some benchmarks there is no way to be precise but we can try to use the examples in place right now.
For instance, take a look at `MSFAA` vs `Update Application Status` and what was defined to them.
It is just a matter to have a "best guess" right now.
The most important, IMO, would be not left as just 1 job to be activated per time. Open to suggestions.